model/src/mpicomm.F90

```diff
@@ -0,0 +1,67 @@
+!> @file
+!> @brief Defines MPI communicator id as constants for global use.
+!>
+!> @author H. L. Tolman  @date 05-Jun-2018
+!>
+#include "w3macros.h"
+
+!>
+!> @brief Define some mpi constants for global use
+!>
+!> @author  J. M. Sexton  @date 19-Jul-2024
+!>
+!
+#ifndef ENDIANNESS
+#define ENDIANNESS "native"
+#endif
+!
+!/ ------------------------------------------------------------------- /
+MODULE MPICOMM
```

Comment:
I'm not sure what the ramifications would be here, I'll consult with one of my collaborators who'd originally suggested having it as a separate file. I believe the intent was to make it more portable and obviously defined. Would [w3parall.F90](https://github.com/NOAA-EMC/WW3/blob/develop/model/src/w3parall.F90) be appropriate? I do see that `IS_ESMF_COMPONENT` is in [constants.F90](https://github.com/NOAA-EMC/WW3/blob/b4e119e18c747fe20d5794ab6dfa33bdc3617aed/model/src/constants.F90#L104-L109)

Reply:
I can move things to another location if that's your preference. When you know, can you point me to where specifically so that I can push an update?

Reply:
w3parall is mostly for unstructured grids, where as this is for all grids, so I think constants might be a better choice.  Others can weigh in on this thought too.

Reply:
Thanks for the explanation, and I'm also interested in others thoughts 🙂  I was originally looking at w3parall because the [SYNCHRONIZE_GLOBAL_ARRAY](https://github.com/NOAA-EMC/WW3/blob/develop/model/src/w3parall.F90#L1516-L1567) has been useful in our coupling tests so far, although I do see now that the header for this file indicates it's "Parallel routines for implicit solver"